desk/app/groups.hoon

```diff
@@ -846,23 +834,26 @@
   ==
 ::
 ++  to-group-ui-2
-  |=  [=net:g =group:g]
+  |=  [=flag:g =net:g =group:g]
   ^-  group-ui:v2:g
   :-  (to-group-2 group)
-  ?+  -.net  ~
-      ::XX make sure this plays well with
-      ::   the frontend
-      %sub  ~
+  ?.  ?=(%sub -.net)  ~
+  =/  =status:neg
+    (read-status:neg bowl [p.flag %groups])
```

Comment:
Note that `+read-status:neg` also does a `.^`... which I think will still crash on you (even though you're scrying into yourself) in scenarios where in the previous event, this agent was suspended, and now it's being brought back online again and runs this migration. (The scry goes into gall, who is trying to make the agent live, but hasn't committed the live agent into its own/arvo's state yet.)

So for filling this in, too, you'll want to defer to the next event.

Reply:
The scries here should be good, since these gates are not used during `+on-load`.

Reply:
You're right, my bad.